operator/k8s_service_sync.go

```diff
@@ -205,12 +205,12 @@ func startSynchronizingServices() {
 	go func() {
 		cache.WaitForCacheSync(wait.NeverStop, svcController.HasSynced)
 		swgSvcs.Stop()
+		swgSvcs.Wait()
+		close(k8sSvcCacheSynced)
```

Comment:
So we don't have anywhere relying on this channel to guarantee that endpoints are now synced?

Reply:
No, I've checked. I also did not change the name of the channel because it says "svc cache synced" not "svc and eps caches synced"